.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,264 @@
+name: PHPUnit
```

Comment:
Better to pick the one from app-tutorial:
https://github.com/nextcloud/app-tutorial/tree/master/.github/workflows